src/components/common/Members/MembersSection.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useCallback, ReactNode } from 'react';
+import classnames from 'classnames';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import MembersList from '~shared/MembersList';
+import LoadMoreButton from '~shared/LoadMoreButton';
+import SortingRow, {
+  Props as SortingProps,
+} from '~shared/MembersList/SortingRow';
+import { Watcher, Contributor } from '~types';
+// import useColonyMembersSorting from '~modules/dashboard/hooks/useColonyMembersSorting';
+
+import styles from './MembersSection.css';
+
+const displayName = 'common.Members.MembersSection';
+
+const MSG = defineMessages({
+  contributorsTitle: {
+    id: `${displayName}.contributorsTitle`,
+    defaultMessage: 'Contributors',
+  },
+  watchersTitle: {
+    id: `${displayName}.watchersTitle`,
+    defaultMessage: 'Watchers',
+  },
+  watchersDescription: {
+    id: `${displayName}.watchersDescription`,
+    defaultMessage: "Members who don't currently have any reputation",
+  },
+  noMembersFound: {
+    id: `${displayName}.noMembersFound`,
+    defaultMessage: 'No members found',
+  },
+});
+
+interface Props {
+  isContributorsSection: boolean;
+  members: Watcher[] | Contributor[];
+  canAdministerComments: boolean;
+  extraItemContent?: (user: (Watcher | Contributor)['user']) => ReactNode;
+  itemsPerSection?: number;
+}
+
+const MembersSection = ({
```

Comment:
I feel like this trying too hard to be both a watcher and a contributors section. They both use the base `MembersList` but have different headings (contributors has the sort functionality), so I think it makes sense to create a `WatchersSection` and a `ContributorsSection`.